### PR TITLE
Inform user when addon is outdated

### PR DESCRIPTION
**Pull request by 3mkay**
Adds a post-start script which checks installed version of addon vs github latest and reports back to user.

### CODE REVIEW

File: scripts/check-addon [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#ddev-generated
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+STARS='**********************************'
+
+ADDONVER=$(ddev add-on list --json-output --installed | jq -r '.raw[] | select(.Name=="annertech-ddev") | .Version')
+LATESTVER=$(ddev exec curl -sL https://api.github.com/repos/annertech/annertech-ddev/releases/latest | ddev exec jq -r \".tag_name\")
```
Comment (bserem) [inline scripts/check-addon:11]: I like that!

**Comment by bserem** [comment]
I see `ddev exec` running on exec-host, although it's post-start. I haven't tested this yet, I will and get back to you.
I like the curl trick, a lot. We shouldn't be far away from getting this done.

**Comment by 3mkay** [comment]
I hate escaping quotes!

Spotted that the first `jq` call is _not_ being made via `ddev` but it's a nightmare of quotes.
Need to look at it again.

**Comment by 3mkay** [comment]
> > I see `ddev exec` running on exec-host, although it's post-start. I haven't tested this yet, I will and get back to you. I like the curl trick, a lot. We shouldn't be far away from getting this done.
> 
> Yeah. I wonder whether it could be turned-around to force it to run on the web container and simplify those commands. I'm only running on the container because we can't be sure that `curl` or `jq` are installed locally.

Looks like we need to run this on the host and call `ddev exec` otherwise we cannot get the version of the installed addon.

**Comment by bserem** [comment]
> > I see `ddev exec` running on exec-host, although it's post-start. I haven't tested this yet, I will and get back to you. I like the curl trick, a lot. We shouldn't be far away from getting this done.
> 
> Yeah. I wonder whether it could be turned-around to force it to run on the web container and simplify those commands. I'm only running on the container because we can't be sure that `curl` or `jq` are installed locally.

It's tricky...
Use `exec` for the web container rather than `exec-host` which always runs on the host. However, you need `ddev` which is only available in the host, so this script needs both the host and the web container.
But you can grep the manifest file from inside the web container:
```
cat .ddev/addon-metadata/annertech-ddev/manifest.yaml | grep -oP '(?<=version: ")[^"]+'
1.14
```

**Comment by bserem** [comment]
image

Pushing some changes to your branch

**Comment by bserem** [comment]
cc @vzsigmond

**Pull request merged**